ui/apps/platform/src/Containers/MainPage/Navigation/HorizontalSubnav.tsx

```diff
@@ -101,7 +104,18 @@ function getSubnavDescriptionGroups(
             },
             {
                 type: 'link',
-                content: 'Virtual Machines',
+                content: () => (
+                    <Flex
+                        direction={{ default: 'row' }}
+                        alignItems={{ default: 'alignItemsCenter' }}
+                        spaceItems={{ default: 'spaceItemsMd' }}
+                    >
+                        <FlexItem>Virtual Machines</FlexItem>
+                        <FlexItem>
+                            <DeveloperPreviewLabel />
+                        </FlexItem>
+                    </Flex>
```

Comment:
Nit, but we could probably achieve the same layout in a less verbose way with:
```tsx
                    <Split hasGutter>
                        <SplitItem>Virtual Machines</SplitItem>
                        <DeveloperPreviewLabel />
                    </Split>
```

Reply:
Yeah, that's valid. With the `FlexItem`, it works the same 👍🏼